db/db.go

```diff
@@ -47,9 +47,10 @@ func createDatabase(dbName string, cmd *exec.Cmd) {
 		log.Panicf("Error running script: %s", err)
 	}
 	connectionString := formConnectionString(dbName)
-	DB, _ = connectToDatabase(connectionString)
-	useDatabaseCommand := "USE " + dbName
-	ExecuteChangeCommand(useDatabaseCommand, "Failed to switch databases")
+	DB, err = connectToDatabase(connectionString)
+	if err != nil {
+		log.Fatalln(err.Error())
```

Comment:
Can we change this to a log.Fatalf with a little message of our own prepended for consistency?